src/WebJobs.Extensions.DurableTask/ActivityTriggerAttribute.cs

```diff
@@ -13,7 +13,12 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{Activity} ({Version})")]
+#if NETSTANDARD2_0
+    [Binding(TriggerHandlesReturnValue = true)]
+#else
     [Binding]
+#endif
```

Comment:
This suggests we're dual compiling for 1.0?  We have a wave of breaking changes coming through (to remove the [Obsolete] attributes), so this may get trickier. You may want to move to a spot where you have 2 separate branches, rather than #ifdef.

Reply:
That's a good point. Right now we have a single NuGet package which targets both .NET 4.6 and .NET Standard 2.0. It's nice because the maintenance is very straightforward. I can certainly see this getting ugly with the new changes you're making. I'll defer that decision to later when I see the scope of the changes. :)